sdcm/nemesis.py

```diff
@@ -2776,9 +2776,13 @@ def disrupt_memory_stress(self):
         no coredump, serious db error occur during the heavy load of memory.
         """
         self._set_current_disruption(f'MemoryStress on {self.target_node}')
-        self.target_node.install_epel()
-        self.target_node.remoter.run('sudo yum install -y stress-ng')
-
+        if self.target_node.distro.is_rhel_like():
+            self.target_node.install_epel()
+            self.target_node.remoter.sudo('yum install -y stress-ng')
+        elif self.target_node.distro.is_ubuntu():
```

Comment:
is_ubuntu() or is_debian() ?

Reply:
didn't check this on debian so can tell that it works on Ubuntu only, anyway we don't have Debian based Machine Image so this makes supporting Debian currently irrelevant

Reply:
You are right.